vnext/Folly/Folly.vcxproj

```diff
@@ -297,6 +318,14 @@
       DestinationFolder="$([MSBuild]::NormalizePath($(FollyDir)..))"
       OverwriteReadOnlyFiles="true" />
   </Target>
+  <Target Name="UnzipFastFloat" BeforeTargets="PrepareForBuild" DependsOnTargets="DownloadFastFloat">
+    <Message Condition="!Exists('$(FastFloatZipDir)\include\fast_float.h')" Importance="High" Text="Unzipping FastFloat to $([MSBuild]::NormalizePath($(FastFloatZipDir)..))." />
+    <Unzip
+      Condition="!Exists('$(FastFloatZipDir)\include\fast_float.h')"
+      SourceFiles="$(FastFloatZipFile)"
+      DestinationFolder="$([MSBuild]::NormalizePath($(FollyDir)..))"
+      OverwriteReadOnlyFiles="true" />
+  </Target>
   <Target Name="WriteCGManifest" BeforeTargets="PrepareForBuild" DependsOnTargets="DownloadFolly" Inputs="$(FollyZipFile)" Outputs="$(CGManifestFile)">
     <PropertyGroup>
       <CGManifestText>{
```

Comment:
The fast_float info/commit should be added here.

Reply:
added!